sections/02_intro.md

```diff
@@ -1,97 +1,155 @@
 ## Introduction
 
-### Potential writing prompt
```

Comment:
> Maybe a patient comes into hospital, gets his molecules measured, grants access to his EHR. Then our deep learning based engine/knowledge-base gives diagnosis/categorization/medication advice. 

I like the idea but I think it would be better suited for short discussion about the future of deep learning in precision medicine "if everything lives up to the hype"